controllers/projects_controller.go

```diff
@@ -190,6 +162,7 @@ func (u *Projects) Update(c web.C, w http.ResponseWriter, r *http.Request) {
 	encoder.Encode(jsonProject)
 }
 
+// TODO: 所属なしのissueはTODOリストではなくnoneリストに突っ込む
```

Comment:
__[golint]__
* [golint] comment on exported method Projects.FetchGithub should be of the form "FetchGithub ..."